src/common/components/game/DraftDeck.tsx

```diff
@@ -0,0 +1,43 @@
+import { sortBy, times } from 'lodash';
+import * as React from 'react';
+
+import * as w from '../../types';
+import { Card } from '../card/Card';
+import CardBack from '../card/CardBack';
+
+interface DraftDeckProps {
+  cards: w.CardInGame[]
+}
+
+export default class DraftDeck extends React.PureComponent<DraftDeckProps> {
+  public render(): JSX.Element {
+    const { cards } = this.props;
+
+    return (
+      <div style={{  }}>
+        {
+          sortBy(cards, ['cost', 'name']).map((card, idx) =>
+            <div
+              key={idx}
+              style={{ float: 'left' }}
```

Comment:
Why float over flex?

Reply:
I still to this day don't understand flex